core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchReaderIterator.java

```diff
@@ -857,11 +857,10 @@ static void doLookup(ClientContext context, String server, Map<KeyExtent,List<Ra
         MultiScanResult scanResult = imsr.result;
 
         if (timer != null) {
-          timer.stop();
           log.trace("tid={} Got 1st multi scan results, #results={} {} in {}",
               Thread.currentThread().getId(), scanResult.results.size(),
               (scanResult.more ? "scanID=" + imsr.scanID : ""),
-              String.format("%.3f secs", timer.scale(SECONDS)));
+              String.format("%.3f secs", timer.elapsed().toMillis() / 1000.0));
```

Comment:
A lot of these immediately call toMillis().  Those could call the method with the time unti

```suggestion
              String.format("%.3f secs", timer.elapsed(TimeUnit.MILLIS) / 1000.0));
```